docs/quickstart.cn.md

```diff
@@ -7,191 +7,157 @@ toc: false
 ```
 
-### 运行HoodieJavaApp
-
-运行 __hudi-spark/src/test/java/HoodieJavaApp.java__ 类，将两个提交(提交1 => 100个插入，提交2 => 100个更新到先前插入的100个记录)放到DFS/本地文件系统上。从命令行运行脚本
+设置表名、基本路径和数据生成器以生成记录。
 
 ```
-cd hudi-spark
-./run_hoodie_app.sh --help
-Usage: <main class> [options]
-  Options:
-    --help, -h
-       Default: false
-    --table-name, -n
-       table name for Hudi sample table
-       Default: hoodie_rt
-    --table-path, -p
-       path for Hudi sample table
-       Default: file:///tmp/hoodie/sample-table
-    --table-type, -t
-       One of COPY_ON_WRITE or MERGE_ON_READ
-       Default: COPY_ON_WRITE
-```
-
-这个类允许您选择表名、输出路径和存储类型。在您自己的应用程序中，请确保包含`hudi-spark`模块依赖并遵循类似模式通过数据源写入/读取数据集。
+import org.apache.hudi.QuickstartUtils._
+import scala.collection.JavaConversions._
+import org.apache.spark.sql.SaveMode._
+import org.apache.hudi.DataSourceReadOptions._
+import org.apache.hudi.DataSourceWriteOptions._
+import org.apache.hudi.config.HoodieWriteConfig._
 
-## 查询Hudi数据集
+val tableName = "hudi_cow_table"
+val basePath = "file:///tmp/hudi_cow_table"
+val dataGen = new DataGenerator
+```
 
-接下来，我们将样本数据集注册到Hive Metastore并尝试使用[Hive](#hive)、[Spark](#spark)和[Presto](#presto)进行查询
+[数据生成器](https://github.com/apache/incubator-hudi/blob/master/hudi-spark/src/main/java/org/apache/hudi/QuickstartUtils.java)
+可以基于行程样本模式生成插入和更新的样本。
+[这里](https://github.com/apache/incubator-hudi/blob/master/hudi-spark/src/main/java/org/apache/hudi/QuickstartUtils.java#L57)
 
-### 本地启动Hive Server
+## 插入数据 {#inserts}
+生成一些新的行程样本，将其加载到DataFrame中，然后将DataFrame写入Hudi数据集中，如下所示。
 
 ```
-hdfs namenode # start name node
-hdfs datanode # start data node
-
-bin/hive --service metastore  # start metastore
-bin/hiveserver2 \
-  --hiveconf hive.root.logger=INFO,console \
-  --hiveconf hive.input.format=org.apache.hadoop.hive.ql.io.HiveInputFormat \
-  --hiveconf hive.stats.autogather=false \
-  --hiveconf hive.aux.jars.path=/path/to/packaging/hudi-hive-bundle/target/hudi-hive-bundle-0.4.6-SNAPSHOT.jar
-
+val inserts = convertToStringList(dataGen.generateInserts(10))
+val df = spark.read.json(spark.sparkContext.parallelize(inserts, 2))
+df.write.format("org.apache.hudi").
+    options(getQuickstartWriteConfigs).
+    option(PRECOMBINE_FIELD_OPT_KEY, "ts").
+    option(RECORDKEY_FIELD_OPT_KEY, "uuid").
+    option(PARTITIONPATH_FIELD_OPT_KEY, "partitionpath").
+    option(TABLE_NAME, tableName).
+    mode(Overwrite).
+    save(basePath);
 ```
 
-### 运行Hive同步工具
-Hive同步工具将在hive Metastore中更新/创建必要的元数据(模式和分区)，这允许模式演变和新分区的增量添加。它使用一种增量方法，将最后一次的同步提交时间存储在TBLPROPERTIES中，并且只同步上次存储的同步提交时间后的提交。
-每次写入后， [Spark Datasource](writing_data.html #datasource-writer)和[DeltaStreamer](writing_data.html＃deltastreamer)都可执行此操作。
+`模式(覆盖)`覆盖并重新创建数据集(如果已经存在)。
+您可以检查在`/tmp/hudi_cow_table/<region>/<country>/<city>/`下生成的数据。我们提供了一个记录键
+([schema](＃sample-schema)中的"uuid")，分区字段("region /county/city"）和组合逻辑("ts"[schema](＃sample-schema))
```

Comment:
“＃”(Unicode?) => "#" in ASCII
and the double quotes (") should be changed to "`" for proper rendering of the strings..